src/presolve.jl

```diff
@@ -239,6 +245,7 @@ function extract_reduced_problem!(ps::PresolveData{T}) where {T}
 
         # Set new column
         pb.acols[jnew] = Col{T}(cind, cval)
+
         pb.var_types[jnew] = ps.pb0.var_types[jold]
```

Comment:
Since we may update variable types, we want to use the up-to-date ones.
```suggestion
        pb.var_types[jnew] = ps.var_types[jold]
```